examples/cli.rs

```diff
@@ -6,8 +6,8 @@ use domain::base::{Dname as DnameO, Message, MessageBuilder, ParsedDname, Rtype}
 use domain::rdata::AllRecordData;
 use log::{info, trace};
 use odoh_rs::*;
+use rand::prelude::*;
 use rand::rngs::StdRng;
-use rand::{Rng, SeedableRng};
```

Comment:
This change is not really needed as long  as the feature been updated above.

Reply:
Reverted.